.github/workflows/test_iree.yml

```diff
@@ -110,3 +110,6 @@ jobs:
         run: |
           source ${VENV_DIR}/bin/activate
           pytest iree_tests/pytorch/models -s -n 4 -k real_weights -rpfE --timeout=1200 --retries 2 --retry-delay 5 --durations=0
+      - name: "Running SDXL pipeline benchmark"
+        run: |
+          bash iree_tests/benchmarks/benchmark_sdxl_cpu.sh
```

Comment:
This may be better suited to a separate workflow file that runs on its own schedule, if the goal is to get regular benchmark numbers.

This file is currently running only when there is pull request activity:
```yml
name: IREE Test Suite
on:
  # TODO(scotttodd): run on schedule (nightly), and/or on pushes to `main`
  pull_request:
    paths:
      - "iree_tests/**"
  workflow_dispatch:
```

Reply:
Especially since this "cpu" script is running under `linux_x86_64_w7900_gpu_models` :P

Reply:
(And how much do we even care to benchmark this model on CPU?)

Reply:
Haha true. The team asked that it would be great to have performance numbers with every PR to make sure we don't regress. But, I think they were referring to rocm backend probably. You think I should just put in seperate area for now then?

Reply:
Ok in this repo, I will create a new scheduled workflow for benchmarking. In IREE repo, I would vote for add this on to the end of the pkgci test jobs, so that at least for this sprint over the next couple of weeks, the dev can easily check accuracy and benchmarking results in the same workflow logs for all sdxl related stuff. (It would just be adding one line right? Because we are cloning this repo there too, just adding
```
- name: "Running SDXL pipeline benchmark"
        run: |
          bash iree_tests/benchmarks/benchmark_sdxl_cpu.sh
```
to the end of the job in that repo workflow file should work)

Reply:
Also because, we won't be able to run a benchmark if we can't compile the submodules and shouldn't be running it if accuracy is off. So, instead of separating them (would need to also duplicate all the compiling steps for the submodels done here), probably best to keep them as a sequential execution)

Reply:
SGTM!